matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/MatrixConfiguration.kt

```diff
@@ -35,7 +36,8 @@ data class MatrixConfiguration(
          * Optional proxy to connect to the matrix servers
          * You can create one using for instance Proxy(proxyType, InetSocketAddress.createUnresolved(hostname, port)
          */
-        val proxy: Proxy? = null
+        val proxy: Proxy? = null,
+        val outboundSessionKeySharingStrategy: OutboundSessionKeySharingStrategy = OutboundSessionKeySharingStrategy.WhenSendingEvent
```

Comment:
Please document this new configuration member

Reply:
Responsibility moved to the application side (RoomDetailViewModel).